src/main/java/com/faforever/client/preferences/Preferences.java

```diff
@@ -89,6 +89,7 @@
   IntegerProperty cacheLifeTimeInDays = new SimpleIntegerProperty(30);
   BooleanProperty gameDataCacheActivated = new SimpleBooleanProperty(false);
   BooleanProperty debugLogEnabled = new SimpleBooleanProperty(false);
+  MapProperty<Integer, String> playerNotes = new SimpleMapProperty<>(observableHashMap()); // key - player ID
```

Comment:
Rather than the comment just name the variable `notesByPlayerId` like the other maps are named.

Also this should go into one of the sub preference objects or a new one. I am trying to move away from having things in the top level object